angular/projects/spark-angular/src/public-api.ts

```diff
@@ -48,14 +48,28 @@ export { SprkLinkComponent } from './lib/components/sprk-link/sprk-link.componen
-export { SprkMastheadSelectorModule } from './lib/components/sprk-masthead/sprk-masthead-selector/sprk-masthead-selector.module';
+export { SprkMastheadAccordionItemComponent } from './lib/components/sprk-masthead/components/sprk-masthead-accordion-item/sprk-masthead-accordion-item.component';
+export { SprkMastheadAccordionItemModule } from './lib/components/sprk-masthead/components/sprk-masthead-accordion-item/sprk-masthead-accordion-item.module';
+export { SprkMastheadAccordionComponent } from './lib/components/sprk-masthead/components/sprk-masthead-accordion/sprk-masthead-accordion.component';
+export { SprkMastheadAccordionModule } from './lib/components/sprk-masthead/components/sprk-masthead-accordion/sprk-masthead-accordion.module';
+export { SprkMastheadSelectorComponent } from './lib/components/sprk-masthead/components/sprk-masthead-selector/sprk-masthead-selector.component';
+export { SprkMastheadSelectorModule } from './lib/components/sprk-masthead/components/sprk-masthead-selector/sprk-masthead-selector.module';
 export { SprkMastheadComponent } from './lib/components/sprk-masthead/sprk-masthead.component';
 export { SprkMastheadModule } from './lib/components/sprk-masthead/sprk-masthead.module';
+export { SprkMastheadNavCollapsibleButtonComponent } from './lib/components/sprk-masthead/components/sprk-masthead-nav-collapsible-button/sprk-masthead-nav-collapsible-button.component';
+export { SprkMastheadNavCollapsibleButtonModule } from './lib/components/sprk-masthead/components/sprk-masthead-nav-collapsible-button/sprk-masthead-nav-collapsible-button.module';
+export { SprkMastheadBrandingDirective } from './lib/components/sprk-masthead/directives/sprk-masthead-branding/sprk-masthead-branding.directive';
+export { SprkMastheadBrandingModule } from './lib/components/sprk-masthead/directives/sprk-masthead-branding/sprk-masthead-branding.module';
+export { SprkMastheadLinkDirective } from './lib/components/sprk-masthead/directives/sprk-masthead-link/sprk-masthead-link.directive';
+export { SprkMastheadLinkModule } from './lib/components/sprk-masthead/directives/sprk-masthead-link/sprk-masthead-link.module';
+export { SprkMastheadNavBarDirective } from './lib/components/sprk-masthead/directives/sprk-masthead-nav-bar/sprk-masthead-nav-bar.directive';
+export { SprkMastheadNavBarModule } from './lib/components/sprk-masthead/directives/sprk-masthead-nav-bar/sprk-masthead-nav-bar.module';
+export { SprkMastheadNavCollapsibleDirective } from './lib/components/sprk-masthead/directives/sprk-masthead-nav-collapsible/sprk-masthead-nav-collapsible.directive';
+export { SprkMastheadNavCollapsibleModule } from './lib/components/sprk-masthead/directives/sprk-masthead-nav-collapsible/sprk-masthead-nav-collapsible.module';
+export { SprkMastheadNavItemDirective } from './lib/components/sprk-masthead/directives/sprk-masthead-nav-item/sprk-masthead-nav-item.directive';
```

Comment:
Do we need to add any of these to SparkAngularModule as well?

Reply:
We do not because they are all imported in the sprk-masthead module. Since these are all namespaced like `SprkMasthead...` the expectation is that they are only used with the Masthead. I think we encountered this question back when RV or you did the masthead selector work and thats where we landed as well.

Reply:
Makes sense. I'll remove some extra stuff from Autocomplete.